Assets/Scripts/CollectibleBomb.cs

```diff
@@ -16,8 +23,22 @@ private void OnTriggerEnter2D(Collider2D collision)
     {
         if (collision.CompareTag("Player"))
         {
-            LevelEvents.Instance.BombCollect.Invoke(color, ServiceLocator.ActiveZoneController.activeZoneName);
-            Destroy(gameObject);
+            if (m_weaponController.BombHandler.CurrentBombColor != color || m_weaponController.BombHandler.AmmoCount < WeaponController.MAX_BOMB_AMMO)
```

Comment:
This check will fail when you collect the first ever bomb and the bomb happens to be not red.

https://gyazo.com/929f25c490c565fdb0a47e10b9f45b06

Reply:
I had that issue. In my level there were 2 overlapping collectibles in that same location. Check if that is the case.

Reply:
It was indeed a duplicate bomb